src/pass.js

```diff
@@ -101,7 +101,8 @@ export class Pass {
   cancel() {
     if (this.scheduled_ != -1) {
       timer.cancel(this.scheduled_);
-      this.scheduled_ = -1;
     }
+    this.scheduled_ = -1;
+    this.nextTime_ = 0;
```

Comment:
No bug, it just didn't make sense to have a `nextTime_` of 10 minutes from now when the pass is cancelled.

I'm actually not sure how this change snuck it, it's supposed to be on a separate branch...